### PR TITLE
Hide hiddenfiles support

### PR DESCRIPTION
**Pull request by dvrajan**

### CODE REVIEW

**Comment by spacecowboy** [comment]
Thanks for the PR @dvrajan 

I'm happy with the idea, but I don't think `AbstractFilePickerFragment.java` should be changed for this. Not all file stores necessarily have the concept of hidden files, whereas the SD-card definitely does.

File: library/src/main/java/com/nononsenseapps/filepicker/FilePickerFragment.java [hunk 1]
```diff
@@ -297,14 +297,18 @@ public void onNewFolder(final String name) {
...
      *
      * @param file to maybe add. Can be either a directory or file.
      * @return True if item should be added to the list, false otherwise
      */
     protected boolean isItemVisible(final File file) {
-        return isDir(file) || (mode == MODE_FILE || mode == MODE_FILE_AND_DIR);
+        if(!showHiddenItems && file.isHidden()){
+            return false;
+        }
+        return (isDir(file) || (mode == MODE_FILE || mode == MODE_FILE_AND_DIR);
```
Comment (spacecowboy) [inline library/src/main/java/com/nononsenseapps/filepicker/FilePickerFragment.java]: Missing parenthesis on the end of this line

**Comment by dvrajan** [comment]
@spacecowboy What you say makes sense. I have moved the toggle to ```FilePickerFragment.java``` now. What do you think?

**Comment by spacecowboy** [comment]
Looks good. I've made an issue to remember to update the samples also: #59

**Comment by tatocaster** [comment]
Is this already in production, because I cant find those changes in projects, added by gradle.

**Comment by spacecowboy** [comment]
@tatocaster It is not uploaded to JCenter yet, but I can do that right now.

**Comment by tatocaster** [comment]
Yes I asked that. I extended fragment and already wrote that functionality but It will be good to have a recent version on JCenter always. Otherwise that project is awesome! 
Thanks 
:+1:

**Comment by spacecowboy** [comment]
@tatocaster __2.5.0__ is now uploaded to JCenter.

[changelog](https://github.com/spacecowboy/NoNonsense-FilePicker/blob/master/CHANGELOG.md)

**Pull request merged**